BeeSwift/ChooseHKMetricViewController.swift

```diff
@@ -36,7 +36,7 @@ class ChooseHKMetricViewController: UIViewController {
             attrString.append(NSMutableAttributedString(string: "\(self.goal.slug)\n",
                 attributes: [NSAttributedStringKey.font: UIFont.beeminder.defaultBoldFont]))
             
-            attrString.append(NSMutableAttributedString(string: "Associate a HealthKit metric with your goal.\nWhile configured as the source for this goal, Apple Health will periodically provide Beeminder for iOS with the corresponding data. Beeminder for iOS will add that data as datapoints to your goal automatically.\nTap a metric from the list below, then tap save to update your goal.",
+            attrString.append(NSMutableAttributedString(string: "HealthKit metric as autodata!\nWith this metric as the autodata source for your goal, Apple Health will keep Beeminder up to date with your latest data automatically.\nTap a metric from the list below, then tap save to update your goal.",
```

Comment:
While we are here, we could consider HealthKit -> Health (the app shows up as Health and not HealthKit; HealthKit is the name of the toolkit the developers use to link to Health/its data).
What do you think?

Reply:
```suggestion
            attrString.append(NSMutableAttributedString(string: "Apple Health as autodata!\nWith this metric as the autodata source for your goal, Apple Health will keep Beeminder up to date with your latest data automatically.\nTap a metric from the list below, then tap save to update your goal.",
```

Reply:
Agreed! I didn't realize that.